docs/api.rst

```diff
@@ -0,0 +1,14 @@
+.. _api:
+
+API Reference
+=============
+
+.. currentmodule:: gordon
+
+main
+----
+
+.. automodule:: gordon.main
+
+.. autofunction:: gordon.main.setup
```

Comment:
These will pull in docstrings defined on the module level of `main` and under `main.setup`.